lib/beaker/dsl/install_utils/puppet_utils.rb

```diff
@@ -693,7 +693,11 @@ def install_puppetagent_dev_repo( host, opts )
           when /^windows$/
             release_path << 'windows'
             onhost_copy_base = '`cygpath -smF 35`/'
-            arch_suffix = arch =~ /64/ ? '64' : '86'
+            should_install_64bit = host.is_x86_64? && !host['install_32'] && !opts['install_32']
```

Comment:
Would prefer this setting to be `:ruby_arch`, which is what we've use in Puppet since the introduction of Ruby 2.0.0 / x64 support (which involved build pipeline changes from @rji)

See https://tickets.puppetlabs.com/browse/PUP-2388 for the history around `:ruby_arch` or the associated PR at https://github.com/puppetlabs/puppet/pull/2903

I've used `:install_32` for consistency now with what I see in https://github.com/puppetlabs/beaker/commit/6f8deb7dbc7bc75b8624df754c261f4d2b58b999 but would prefer to see this changed to `:ruby_arch` if we know we don't have downstream consumers here.

If you look at https://github.com/puppetlabs/puppet/tree/master/acceptance/config/nodes you'll see all the existing Windows node definitions use `:ruby_arch`


/cc @joshcooper @justinstoller @kevpl @anodelman